src/Package/MSBuild.VSSetup/files.swr

```diff
@@ -9,6 +9,16 @@ vs.dependencies
   vs.dependency id=Microsoft.VisualStudio.PackageGroup.NuGet
                 version=[15.0,17.0)
 
+vs.relatedProcessFiles
+  vs.relatedProcessFile Path="[InstallDir]\MSBuild\Current\Bin\Microsoft.Build.dll"
```

Comment:
Should we also add MSBuild.exe/dll, Microsoft.Build.Conversion.Core.dll, Microsoft.Build.Engine.dll, and MSBuildTaskHost.exe?

Reply:
The `.exe`s are handled automatically by Setup logic (it already scans for _programs_ under the VS install directory).

The deprecated assemblies I'm less sure about. I _hope_ no one is using them, and MSBuildLocator doesn't help people load them. So I'm not sure how to trade off the cost of scanning for them (it is supposedly nontrivial--which is why we have to use this explicit list instead of having Setup just do "everything under VS") vs. the likelihood of them actually being in use by non-VS programs (hopefully none? But I don't have numbers).

I'd rather do this now and we can always change it later if we find more users.